nvm.sh

```diff
@@ -1407,10 +1407,11 @@ nvm() {
         command rm -f "$NVM_DIR/current" && ln -s "$NVM_VERSION_DIR" "$NVM_DIR/current"
       fi
       if nvm_is_iojs_version "$VERSION"; then
-        echo "Now using io.js $(nvm_strip_iojs_prefix "$VERSION")"
+        echo -n "Now using io.js $(nvm_strip_iojs_prefix "$VERSION")"
       else
-        echo "Now using node $VERSION"
+        echo -n "Now using node $VERSION"
```

Comment:
`echo -n` isn't supported in `sh` (although it is supported in all other shells. Instead of using that, could we capture the npm version in a variable before the if/else, and interpolate that into a single `echo` command?